src/app/settings/admin/presets/theme.scss

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@mixin theme-preset-list-component($colors) {
+  .mat-card-content {
+    button {
+      color: inherit;
```

Comment:
This could be moved to the component style I think as there is no color value used.

Reply:
It was done intentionally as I try not to mix `color`, `background-color`, or any other property that is related to theming with regular style files.